src/main/java/com/horus/yoga/controller/UserController.java

```diff
@@ -26,14 +26,14 @@ public APIResponse<UserDTO> getById(@PathVariable UUID id) {
         return APIResponse.ok(user);
     }
 
-    @GetMapping("/")
+    @GetMapping
     public APIResponse<List<UserDTO>> getAll() {
         List<UserDTO> users = userService.getAll();
         return APIResponse.ok(users);
     }
 
 
-    @PostMapping("/")
+    @PostMapping
```

Comment:
This is required, since you need to send data after the slash.
I do agree with the getAll, but for post the slash NEEDs to be there, please add it so i can accept this PR